rootfs/etc/init.d/atlantis.sh

```diff
@@ -82,6 +82,12 @@ if [ "${ATLANTIS_ENABLED}" == "true" ]; then
 	# Do not export these as Terraform environment variables
 	export TFENV_BLACKLIST="^(AWS_ACCESS_KEY_ID|AWS_SECRET_ACCESS_KEY|AWS_SECURITY_TOKEN|AWS_SESSION_TOKEN|ATLANTIS_.*|GITHUB_.*)$"
 
+    # Configure Git credentials for atlantis to allow access to GitHub private repos (see rootfs/usr/local/bin/git-credential-github)
+    # https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage
+    export GITHUB_USER=${ATLANTIS_GH_USER}
+    export GITHUB_TOKEN=${ATLANTIS_GH_TOKEN}
+    git config --global credential.helper 'github'
```

Comment:
I think the name of the helper should be the script name, no?

Reply:
it's the last part after `git-credential-`
https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage